renormalizer/vibronic/tests/test_pyr4.py

```diff
@@ -214,6 +212,11 @@ def test_pyr_4mode(multi_e, translator):
     mpo = Mpo(mol_list2)
     logger.info(f"mpo_bond_dims:{mpo.bond_dims}")
     mps = Mps.hartree_product_state(mol_list2, condition={"e_1": 1})
+    if dvr:
+        for dof in mol_list2.v_dofs:
+            idx = order[dof]
+            mps[idx] = np.einsum("abc, bd -> adc", mps[idx], basis[idx].dvr_v)
```

Comment:
These lines of code remind of the idea that maybe we should put the information on what "ground state" is defined in the`Basis` class.
In this example, `BasisSHO` have different representations for "ground state" regarding whether DVR is enabled or not.
And in `hartree_product_state`, for dofs that are not specified in `condition`, the default value is set to the "ground state" provided by the basis class.

Reply:
Actually, when DVR is considered, there is a rotation between the basis SHO and SHO_DVR. 
If you only would like to get a hartree_product_state, I think it is easier to just modify `hartree_product_state`. If you'd like to modify BasisSHO,  not only "ground state" but also "1st/2nd/3rd state" should be implemented, because `condition` could be `{"v_1":1}`

Reply:
Makes sense to me. In `hartree_product_state` it should be allowed to specify the state (a vector) instead of just a number.

Reply:
Agree.